### DISCUSSION TITLE
FnB time series, kesalahan frekuensi?

### DISCUSSION THREAD

**Discussion by jonathanariel** [teamalgoritma/community#329] [Q&A] [answered]
Sore, saya sedang mengerjakan capstone ML case FnB, saya sudah melakukan aggregasi dan padding data dan menghasilkan data hourly berikut:

![image](https://user-images.githubusercontent.com/107382811/191700533-f9372ca4-ed33-4c3c-8dc5-df652ca2e60a.png)

Tapi, setelah di ubah menjadi time series dengan ts(), hasil autoplotnya jadi seperti berikut:
![image](https://user-images.githubusercontent.com/107382811/191701141-8bf1c928-3ea7-4c0d-a73e-4b0cda46c099.png)

Kira-kira ada bagian mana yang salah ya? Kenapa tahunnya bisa sampai 2028?

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @jonathanariel ,

Sebelumnya kami ingin memastikan bagaimana cara Anda melakukan proses agregasi untuk mendapatkan data pengujung per jam. Berhubung angka yang Anda dapatkan terlalu besar. Anda bisa mempertimbangkan untuk melakukan agregasi data untuk menghitung jumlah `receipt_number` per jamnya.

Setelah itu, silahkan lakukan filter data berhubung outlet hanya buka dari pukul 10:00 - 22:00. Setelah melakukan filtering data, Anda dapat mencoba untuk dekomposisi untuk melihat pola hariannya (sehingga frequency adalah berapa jam outlet buka dalam 1 hari). Jika dekomposisi satu seasonal dirasa kurang cukup. Anda dapat menggunakan multiple seasonality dengan referensi yang dapat ditemukan di [artikel ini](https://algotech.netlify.com/blog/multiple-seasonal/).

Sekian jawaban dari kami. Anda dapat reply jawaban ini jika terdapat kendala lebih lanjut.
Terima kasih

**Reply by jonathanariel** [reply]
iya kak, ternyata saya salah menangkap pertanyaannya. Sudah di revisi saat sesi mentoring kemarin, terima kasih atas replynya.
